### DISCUSSION TITLE
Compositing

### DISCUSSION THREAD

**Discussion by qustavo** [charmbracelet/lipgloss#470] [Ideas]
I'm trying to build an app that displays a popup which needs to be drawn on top of other view (bubble component). Is it possible to achieve that with lipgloss?

**Comment by meowgorithm** [comment]
Hi! This isn't possible with Lip Gloss quite yet but something we're working on (it's a fairly large endeavor). We'll keep this issue updated with progress.

Related: https://github.com/charmbracelet/bubbletea/issues/79

**Comment by qustavo** [comment]
Is this something to be implemented in lipgloss or somewhere else?

**Comment by meowgorithm** [comment]
It’ll be implemented in Lip Gloss, most likely with additional (and optional) interactive integration in Bubble Tea.

**Comment by qustavo** [comment]
Thanks! Feel free to close this

**Comment by pnsvk** [comment]
can you please post the updates (if any) on this ? its more than 2 years now.

**Comment by bashbunni** [comment]
Hey @pnsvk, looks like @meowgorithm replied in the Bubble Tea compositing issue. Here's a link to that message for anyone else who stumbles across this. https://github.com/charmbracelet/bubbletea/issues/309#issuecomment-2322207775
